src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMNavigator.java

```diff
@@ -62,13 +63,32 @@
     private final String apiUri;
     private String pattern = ".*";
 
+    private @CheckForNull String includes;
+    private @CheckForNull String excludes;
+
     @DataBoundConstructor public GitHubSCMNavigator(String apiUri, String repoOwner, String scanCredentialsId, String checkoutCredentialsId) {
         this.repoOwner = repoOwner;
         this.scanCredentialsId = Util.fixEmpty(scanCredentialsId);
         this.checkoutCredentialsId = checkoutCredentialsId;
         this.apiUri = Util.fixEmpty(apiUri);
     }
 
+    public @Nonnull String getIncludes() {
+        return includes != null ? includes : DescriptorImpl.defaultIncludes;
+    }
+
+    @DataBoundSetter public void setIncludes(@Nonnull String includes) {
+        this.includes = includes.equals(DescriptorImpl.defaultIncludes) ? null : includes;
+    }
+
+    public @Nonnull String getExcludes() {
```

Comment:
@jglick have you ever think that it only your personal style that everybody forced to use because it's FOSS project that supposed to be open and used by everybody and even claimed like jenkins future? 

> could become a type qualifier annotation in the foreseeable future.

I think if it happens, then everybody would automatically start using it, but now it sounds weird.
In this case you hopefully placed `public` before and people that know java and doesn't know this sources and special annotations may interrupt their eyes on known key word. Let's look at `@DataBoundSetter public void setIncludes(`. How do you expect new developers to read this code? Interrupting eyes on every line?